controllers/components/modelcontroller/modelcontroller_actions.go

```diff
@@ -44,7 +44,9 @@ func initialize(_ context.Context, rr *odhtypes.ReconciliationRequest) error {
 		nimState = mc.Spec.Kserve.NIM.ManagementState
 	}
 	extraParamsMap := map[string]string{
-		"nim-state": strings.ToLower(string(nimState)),
+		"nim-state":        strings.ToLower(string(nimState)),
+		"kserve":           strings.ToLower(string(mc.Spec.Kserve.ManagementState)),
+		"modelmeshserving": strings.ToLower(string(mc.Spec.ModelMeshServing.ManagementState)),
```

Comment:
should this have a `-state` suffix ?

Reply:
@ederign / @VedantMahabaleshwarkar how you want the key to be called? 
i am fine with `kserve-state/ modelmeshserving-state`

Reply:
updated